### PR TITLE
fix: add velero support

### PR DESCRIPTION
**Pull request by jstrachan**

### CODE REVIEW

**Comment by jstrachan** [comment]
I think this needs to wait for a new `jx` binary that defaults an empty map for `velero`

File: systems/velero/values.tmpl.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      config:
+        region: {{ .Requirements.cluster.region | quote }}
+{{- else if eq .Requirements.cluster.provider "azure" }}
+  configuration:
+    provider: azure
+    backupStorageLocation:
+      name: azure
+      bucket: {{ .Requirements.storage.backup.url | removeScheme | quote }}
+      config:
+        storageAccount: {{ .Requirements.veloro.serviceAccount | quote }}
```
Comment (patrickleet) [inline systems/velero/values.tmpl.yaml]: there is a typo here `veloro` instead of `velero`

Comment (jstrachan) [reply systems/velero/values.tmpl.yaml]: many thanks for the catch!

**Pull request closed without merging**